.github/workflows/lint.yaml

```diff
@@ -0,0 +1,87 @@
+name: "Lint"
+
+on: [push, pull_request]
+
+env:
+    fail-fast: true
+
+jobs:
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: PHP-CS-Fixer
+              uses: docker://oskarstark/php-cs-fixer-ga
+              with:
+                  args: --diff --dry-run
+
+    linters:
+        name: Linters
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                allow-failures: [false]
+                operating-system: ['ubuntu-latest']
+                php-version: ['7.4']
+
+        steps:
+            - name: "Checkout code"
+              uses: actions/checkout@v2.3.3
+
+            - name: "Install PHP with extensions"
+              uses: shivammathur/setup-php@2.7.0
+              with:
+                  coverage: "none"
+                  extensions: intl, mbstring, pdo_sqlite
+                  php-version: ${{ matrix.php-version }}
+                  tools: composer:v2
+
+            - name: "Set composer cache directory"
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: "Cache composer"
+              uses: actions/cache@v2.1.2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-${{ matrix.php-version }}-composer-${{ hashFiles('composer.json') }}-allow-failures-${{ matrix.allow-failures }}
+                  restore-keys: ${{ runner.os }}-${{ matrix.php-version }}-composer-
+
+            - name: "Require symfony/flex"
+              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex dev-main
+
+            - name: "Composer update"
+              run: composer update
+
+            - name: Lint YAML files
+              run: |
+                  ./bin/console lint:yaml config --parse-tags
+
+            - name: Lint Twig templates
+              run: |
+                  ./bin/console lint:twig templates --env=prod
+
+            - name: Lint XLIFF translations
+              run: |
+                  ./bin/console lint:xliff translations
```

Comment:
I think there is something non optimal here: as the composer update above will be executed with cache-warmers (due to the `cache:clear` command called after an update), it is likely to fail before reaching these linters steps and we'll get an incomplete run.

A solution to circumvent this could be to run the update without scripts.

Reply:
Good catch! I've added the `--no-scripts` to `composer update`. Thanks!